dags/analyzer_helper/tests/integration/test_discord_load_transformed_members.py

```diff
@@ -0,0 +1,50 @@
+import unittest
+
+from dags.analyzer_helper.discord.discord_load_transformed_members import DiscordLoadTransformedMembers
+from dags.hivemind_etl_helpers.src.utils.mongo import MongoSingleton
+
+
+class TestDiscordLoadTransformedMembers(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.client = MongoSingleton.get_instance().client
+        cls.platform_id = 'discord_platform'
+        cls.db = cls.client[cls.platform_id]
+        cls.collection = cls.db['rawmembers']
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.collection.delete_many({})
+        cls.client.close()
+
+    def setUp(self):
+        self.collection.delete_many({})
+
+    def test_load_recompute_true(self):
+        """
+        Tests that load replaces all existing data when recompute is True
+        """
+        initial_data = [{'_id': '1', 'data': 'initial_data_1'}, {'_id': '2', 'data': 'initial_data_2'}]
+        self.collection.insert_many(initial_data)
+
+        processed_data = [{'_id': '3', 'data': 'processed_data_1'}, {'_id': '4', 'data': 'processed_data_2'}]
```

Comment:
The processed data should match the format in the [link](https://rndadocs.notion.site/Analyzer-Discord-Pipeline-5009512d82a243deb35b3388e17b29c5#:~:text=%7B%0A%09%22id%22%3A%20int%2C%20//%20discord%20id%0A%09%22is_bot%22%3A%20bool%2C%0A%09%22left_at%22%3A%20datetime%2C%0A%09%22joined_at%22%3A%20datetime%2C%0A%09%22options%22%3A%20dict%2C%0A%7D)